gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/db/FabricDatabase.java

```diff
@@ -174,9 +174,9 @@ Licensed to the Apache Software Foundation (ASF) under one or more
   private volatile boolean active;
 
   /** DOCUMENT ME! */
-  private AuthenticationService authenticationService;
+  private AuthenticationServiceBase authenticationService;
```

Comment:
In all the change in this file you have just used the abstract "AuthenticationServiceBase" instead of the interface "AuthenticationService". Any specific reason?

Reply:
Callers now use the getBootCredentials which is not in the interface.